PIL/TiffTags.py

```diff
@@ -130,16 +130,17 @@ def lookup(tag):
     324: ("TileOffsets", LONG, 0),
     325: ("TileByteCounts", LONG, 0),
 
+    330: ("SubIFD", SHORT, 1),
```

Comment:
#6569 added this tag, but as `LONG`, which matches https://web.archive.org/web/20240329145255/https://www.awaresystems.be/imaging/tiff/tifftags/subifds.html